src/hyperion/parameters/tests/test_external_parameters.py

```diff
@@ -49,6 +49,39 @@ def test_beamline_parameters():
     assert params["beamLineEnergy__adjustSlits"] is False
 
 
+def test_i03_beamline_parameters():
+    params = GDABeamlineParameters.from_file(
+        "src/hyperion/parameters/tests/test_data/i04_beamlineParameters"
+    )
+    assert params["flux_predict_polynomial_coefficients_5"] == [
+        -0.0000707134131045123,
+        7.0205491504418,
+        -194299.6440518530,
+        1835805807.3974800,
+        -3280251055671.100,
+    ]
+
+
+def test_parse_exception_causes_warning():
```

Comment:
We can assert that the logger raises a warning using patch

```suggestion
@patch("hyperion.parameters.beamline_parameters.LOGGER")
def test_parse_exception_causes_warning(mock_logger):
    params = GDABeamlineParameters.from_file(
        "src/hyperion/parameters/tests/test_data/bad_beamlineParameters"
    )
    assert params["flux_predict_polynomial_coefficients_5"] == [
        -0.0000707134131045123,
        7.0205491504418,
        -194299.6440518530,
        1835805807.3974800,
        -3280251055671.100,
    ]
    mock_logger.warning.assert_called_once()

```